src/BinaryParsers/ElfBinary/ElfBinary.cs

```diff
@@ -56,7 +56,8 @@ public ElfBinary(Uri uri, string localSymbolDirectories = null) : base(uri)
                 PublicSymbols = publicSymbols;
                 SectionRegions = ELF.Sections.Where(s => s.LoadAddress > 0).OrderBy(s => s.LoadAddress).ToArray();
 
-                CompilationUnits = DwarfSymbolProvider.ParseCompilationUnits(this, DebugData, DebugDataDescription, DebugDataStrings, NormalizeAddress);
+                CompilationUnits = DwarfSymbolProvider.ParseAllCompilationUnits(this, DebugData, DebugDataDescription, DebugDataStrings, NormalizeAddress);
+                CommandLineInfos = DwarfSymbolProvider.ParseAllCommandLineInfos(CompilationUnits);
```

Comment:
>CommandLineInfos [](http://example.com/codeflow?start=16&length=16)

should we make lazyload?
just like PE/Pdb?

Reply:
got it, will check

Reply:
changed to Lazy<> similar to PE.pdb